src/Components/Artwork/Details.tsx

```diff
@@ -89,16 +90,71 @@ export class Details extends React.Component<Props, null> {
     }
   }
 
-  render() {
+  saleInfoLine() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inClosedAuction = sale && sale.is_auction && sale.is_closed
+
     return (
       <div>
-        {this.artistLine()}
-        {this.titleLine()}
-        {this.partnerLine()}
-        {this.props.showSaleLine && this.saleLine()}
+        <div>
+          {inClosedAuction ? "Bidding closed" : this.saleMessageOrBidInfo()}
+          {!inClosedAuction && this.auctionInfo()}
+        </div>
       </div>
     )
   }
+
+  saleMessageOrBidInfo() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inRunningAuction = sale && sale.is_auction && !sale.is_closed
+
+    if (inRunningAuction) {
+      if (sale && sale.is_open) {
+        const sa = artwork.sale_artwork
+        const bids = sa && sa.counts && sa.counts.bidder_positions
+        if (bids && bids > 0) {
+          return sa.highest_bid.display
+        } else {
+          return sa.opening_bid.display
+        }
+      }
+    } else {
+      return artwork.sale_message
+    }
+  }
```

Comment:
Would be great to have a test here but no worries if setup is a blocker. Let me know

Reply:
Also thinking to possibly move this up to MP.

Reply:
Yeah it would be better there (and also simpler to test)